pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -0,0 +1,2436 @@
+    trigger: true
+  - put: performance-tests-docker-image-ci
+    on_failure: *slack_failure_alert_ci
+    params:
+      build: performance-tests-master
+      tag_file: performance-tests-master/.git/ref
+      tag_as_latest: true
+    get_params:
+      save: true
+  - put: performance-tests-docker-image-gcr
+    on_failure: *slack_failure_alert_gcr
+    params:
+      build: performance-tests-master
+      cache_from:
+        - performance-tests-docker-image-ci
+      tag_file: performance-tests-master/.git/ref
+      tag_as_latest: true
+    get_params:
+      skip_download: true
+
+
+# CI Deployments
+- name: "CI Deploy Infrastructure"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure]
+  max_in_flight: 1
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+  - get: census-rm-kubernetes-dependencies-repo
+    trigger: true
+  - get: census-rm-kubernetes-repo
+  - *slack_started_alert_ci
+  - task: "Build & Deploy Infrastructure"
+    on_success: *slack_success_alert_ci
+    on_failure: *slack_failure_alert_ci
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: google/cloud-sdk
+      params:
+        ADMIN_SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+        AUTO_APPLY: true
+        DEV_DUMMY_PGP: true
+        DEV_PUBSUB: true
+        DISABLE_CI_BINDING: true
+        ENV: ((ci-gcp-environment-name))
+        GOOGLE_APPLICATION_CREDENTIALS: /root/gcloud-service-key.json
+        KEEP_TMP_K8S: true
+        SKIP_DEPLOYMENTS: true
+      inputs:
+        - name: census-rm-terraform
+        - name: census-rm-kubernetes-repo
+      run:
+        path: bash
+        args:
+          - -exc
+          - |
+            apt-get install -y unzip procps  # NB: procps is used by Helm
+            git clone https://github.com/kamatama41/tfenv.git ~/.tfenv && \
+            ln -s /root/.tfenv/bin/* /usr/local/bin
+            cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+            $ADMIN_SERVICE_ACCOUNT_JSON
+            EOL
+            gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+            cp -R census-rm-kubernetes-repo/ census-rm-terraform/tmp_rm_kube
+            cd census-rm-terraform
+            tfenv install
+            curl -L https://git.io/get_helm.sh | bash && \
+            helm init --client-only && \
+            helm plugin install https://github.com/rimusz/helm-tiller
+            ./apply.sh
+
+
+- name: "CI Deploy Action-Scheduler"
+  serial: true
+  serial_groups: [ci-deploy-infrastructure,
+                  ci-acceptance-tests]
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+    passed: ["CI Deploy Infrastructure"]
+  - get: census-rm-kubernetes-microservices-repo
+    trigger: true
+  - get: action-scheduler-master
+    trigger: true
+    passed: ["Build Action Scheduler Latest"]
+  - get: action-scheduler-docker-image-gcr
+    params:
+      skip_download: true
```

Comment:
The change to the deployment jobs is to get and trigger from the github repo resource instead of the docker image.